apps/storybook-react/stories/Progress Indicators/Dots.stories.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { DotProgress, Typography } from '@equinor/eds-core-react'
+import styled from 'styled-components'
+
+const Body = styled.div`
+  display: grid;
+  grid-template-rows: min-width;
+  padding: 32px;
+  padding-bottom: 8rem;
+  grid-gap: 2rem;
+`
+
+const Wrapper = styled.div`
+  margin-left: 16px;
+`
+
+const Background = styled.div`
+  background-color: grey;
+  width: 36px;
+  padding: 8px;
+`
+
+export default {
+  title: 'Components|Progress Indicators/Dots',
+  component: DotProgress,
+}
+
+export const Variants = () => {
+  return (
+    <Body>
+      <Typography variant="h2">White</Typography>
```

Comment:
If you want to keep the typography you had, just use the as-attribute

```suggestion
      <Typography variant="h4" as="h2">White</Typography>
```